src/niftyone/__init__.py

```diff
@@ -3,6 +3,11 @@
 from ._version import __version__, __version_tuple__
 
 # Register existing generators
+from .figures.dwi import (
```

Comment:
Just a note for the future that we will at some point need a proper plugin mechanism for user figures.